CHANGELOG/CHANGELOG-6.1.md

```diff
@@ -1,7 +1,36 @@
-# Release notes for v6.1.0
+# Release notes for v6.1.1
 
 [Documentation](https://kubernetes-csi.github.io)
 
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- Update deployment tags to v6.1.0 (#766, @RaunakShah)
+
+### Uncategorized
+
+- CVE fixes: CVE-2023-44487,  CVE-2023-3978 (#929, @dannawang0221)
```

Comment:
Can you run release-notes with the "--markdown-links" flag?  This way it will provide a markdown link.